src/components/20-molecules/file-upload/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 5.0.0
+
+- New property `accessOriginalFile` to choose between compromised and original files. #2036
```

Comment:
What about the name `compressImages` - which describe exactly what it do.

Reply:
Compressing an image should be the default behaviour. With `compressImages` the user would get the original files per default.

Reply:
Yes, default could be true. The user could set compressImages to false to get the original ones.

Reply:
In my opinion default should be compressed. Accessing the original files is an addon for me.

Reply:
Anders gesagt: Der Name accessOriginalFile erklärt nicht genau was passiert. Wenn accessOriginalFile gesetzt ist wird ja NICHT komprimiert. Daher würde ich das umbenennen. Der User muss dann z.b. "compressImages=false" setzen wenn er die originalen will ohne kompression

Reply:
Die Images werden ja trotzdem noch für die Previews komprimiert. Daher wäre `compressImages` nicht passend. Und wegen dem bereits erwähnten default Verhalten finde ich es ebenfalls keine gute Idee.
Meiner Meinung nach beschreibt die Property `accessOriginalFiles` sehr gut was mit ihr gemacht werden kann. Ich habe die changelog Meldung ein Bisschen angepasst.
Einzige alternative wäre für mich `accessUncompressedFiles`. Was haltest du davon?